### PR TITLE
Series calculator

### LINKED ISSUE #153: Provide an active series estimate during check creation
We should give an indication of how many active series a check will generate when creating.

### PR DESCRIPTION
**Pull request by rdubrock**
What is this for?:
It provides customers with an approximation of how much usage they will be generating with a specific check. Closes #153 

series calculator

### CODE REVIEW

File: src/checkUsageCalc.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  checksPerMonth: number;
+  activeSeries: number;
+  logsGbPerMonth: number;
+}
+
+enum CheckSeries {
+  PING = 76,
+  HTTP = 128,
+  DNS = 79,
+  TCP = 59,
```
Comment (leventebalogh) [inline src/checkUsageCalc.ts:19]: **Question:** probably a dumb question, but what do these numbers mean exactly?

File: src/checkUsageCalc.ts [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    case CheckType.TCP:
+      return CheckSeries.TCP;
+    case CheckType.DNS:
+      return CheckSeries.DNS;
+    case CheckType.HTTP:
+      return CheckSeries.HTTP;
+  }
+};
+
+const getMonthlyChecks = (frequencySeconds: number) => {
```
Comment (leventebalogh) [inline src/checkUsageCalc.ts:35]: **Nitpick:** I would probably call it `getChecksPerMonth()` instead as we are referring like that to its return value everywhere else.

**Review by leventebalogh — APPROVED** [review]
Left a few nitpick comments, but generally looks good to me.

File: src/checkUsageCalc.ts (hunk 1 above)
Comment (nathanrodman) [reply src/checkUsageCalc.ts:19]: These numbers are the number of active series generated per check. Different types of checks generate different numbers of metrics

Comment (leventebalogh) [reply src/checkUsageCalc.ts:19]: Thanks for the explanation Nathan 👍

**Pull request merged**